src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -63,7 +63,7 @@ public class SubscriptionWorker<T> : IAsyncDisposable, IDisposable where T : cla
         /// </summary>
         public event AfterAcknowledgmentAction AfterAcknowledgment;
 
-        internal event Action OnEstablishedSubscriptionConnection;
+        public event Action OnEstablishedSubscriptionConnection;
```

Comment:
should stay internal

Reply:
You need to add that project here
https://github.com/ravendb/ravendb/blob/v5.2/src/CommonAssemblyInfo.Linux.cs
and here
https://github.com/ravendb/ravendb/blob/v5.2/src/CommonAssemblyInfo.Windows.cs

Reply:
Actually I'd say that would be valuable to have public. Seeing just the errors the user cannot be sure it was able to connect in the end or not if not new documents are coming. I think we should make it public.